api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -880,4 +880,22 @@ public PersonAddress build() {
 			return personAddress;
 		}
 	}
+
+	/**
+	 * @see Person#setDeathDate(Date)
+	 * @verifies set dead flag depending on deathDate presence
+	 */
+	@Test
+	public void setDeathDate_shouldSetDeadFlagDependingOnDeathDatePresence() {
+		Calendar birthdate = Calendar.getInstance();
+		birthdate.set(1990, Calendar.JUNE, 2);
+		Calendar deathDate = Calendar.getInstance();
+		deathDate.set(2000, Calendar.JUNE, 3);
+		Person person = new Person();
+		person.setBirthdate(birthdate.getTime());
```

Comment:
Since you did not claim the ticket as advised on that link, and some one else did, it means that the one who did so is the one whose pull request is going to be merged. 
I would advice you to choose another ticket, but this time remember to follow the instructions on that page. 😊